projects/storefrontlib/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -105,6 +115,15 @@ export class TabParagraphContainerComponent
     }
   }
 
+  getRegionLabel(): void {
```

Comment:
We can use an async pipe if we can move this to the template somehow which would be a lot cleaner.

Reply:
Given that I don't know how to add SR context to ng-template, I am not sure how to go on about this